src/SFA.DAS.ApplyService.Data/Queries/OversightReviewQueries.cs

```diff
@@ -108,6 +108,114 @@ public async Task<CompletedOversightReviews> GetCompletedOversightReviews(string
+                {
+                    searchString = $"%{searchTerm}%",
+                    gatewayReviewStatusPass = GatewayReviewStatus.Pass,
+                    gatewayReviewStatusFail = GatewayReviewStatus.Fail,
+                    GatewayReviewStatusRejected = GatewayReviewStatus.Rejected,
+                    assessorReviewStatusApproved = AssessorReviewStatus.Approved,
+                    assessorReviewStatusDeclined = AssessorReviewStatus.Declined,
+                    financialReviewStatusApproved = FinancialReviewStatus.Pass,
+                    financialReviewStatusDeclined = FinancialReviewStatus.Fail,
+                    financialReviewStatusExempt = FinancialReviewStatus.Exempt,
+                    applicationStatusRemoved = ApplicationStatus.Removed
+                })).ToList();
+
+                return new PendingAppealOutcomes
+                {
+                    Reviews = reviews
+                };
+            }
+        }
+
+
+        public async Task<CompletedAppealOutcomes> GetCompletedAppealOutcomes(string searchTerm, string sortColumn, string sortOrder)
+        {
+            using (var connection = GetConnection())
+            {
+                var orderByClause = $"{GetSortColumnForAppealOutcome(sortColumn)} { GetOrderByDirection(sortOrder)}";
+
+                var reviews = (await connection.QueryAsync<CompletedAppealOutcome>($@"SELECT 
+                            apply.ApplicationId AS ApplicationId,
+                            apply.ApplicationStatus,
+                            org.Name AS OrganisationName,
+                            apply.GatewayReviewStatus,
+                            apply.FinancialReviewStatus,
+                            apply.ModerationStatus AS ModerationReviewStatus,
+                            apply.UKPRN,
+                            REPLACE(JSON_VALUE(apply.ApplyData, '$.ApplyDetails.ProviderRouteName'),' provider','') AS ProviderRoute,
+                            JSON_VALUE(apply.ApplyData, '$.ApplyDetails.ReferenceNumber') AS ApplicationReferenceNumber,                          
+                            apply.createdat AS ApplicationSubmittedDate,
+                            appel.AppealSubmitedDate AS AppealSubmitedDate,
+                            apply.ApplicationDeterminedDate AS ApplicationDeterminedDate
+                                FROM Apply apply
+                            INNER JOIN Organisations org ON org.Id = apply.OrganisationId
+                            LEFT JOIN OversightReview r ON r.ApplicationId = apply.ApplicationId
+                            LEFT JOIN Appeal appel ON appel.ApplicationId = apply.ApplicationId
+                            WHERE apply.DeletedAt IS NULL
+                            AND ( @searchString = '%%' OR apply.UKPRN LIKE @searchString OR org.Name LIKE @searchString )
+                            and r.Status is null
+                            and appel.Status not in (0,1)
```

Comment:
can you take out 0,1 and inject in variables to represent the appeal statuses you're checking against